packages/protocol/scripts/bash/revoke_contracts.sh

```diff
@@ -0,0 +1,68 @@
+#! /usr/bin/env bash
+set -euo pipefail
+rejects=()
+
+if [ "$#" -ne 1 ]; then
+  echo "Expected usage: ./revoke_contracts.sh contracts.txt"
+  echo "contracts.txt should include contracts you'd like to revoke and refund, one address per line"
+  exit 1
+fi
+
+for contract in $(<"$1"); do
+  echo "Contract $contract"
+
+  # Check that account has been created
+  accountExists=1
+  celocli lockedgold:show "$contract" > /dev/null 2>&1 || accountExists=0
+  if [ $accountExists -eq 0 ]; then
+    echo "Account not created, rejecting $contract"
```

Comment:
Do we actually need to reject here? If they didn't create an account that just means that they never locked any CELO, and so we *should* be able to skip straight to revoking, right?